tests/functional/search-auth.js

```diff
@@ -198,6 +198,34 @@ registerSuite("Search (auth)", {
         .end();
     },
 
+    "Label have correct URL after search without results"() {
+      // load up a garbage search, so we can more easily detect when
+      // the search values we want are loaded.
+      return FunctionalHelpers.openPage(
+        this,
```

Comment:
This is a weird behaviour I ran into from time to time (UX wise). After fixing the lint issue, I forget to add the fixed file and push the commit without the fix. Maybe something like `npm run lint:js && git add .` would fix this?

Reply:
Ah, interesting.

I finally just configured my text editor to auto-prettier all JS files on save, because I had done the same things as well. I'd be a little nervous about making `npm run lint:js && git add .` a default command, frequently I have a lot of unstaged changes that I want to include in different commits.